### DISCUSSION TITLE
Clear conversation? What is the best way?

### DISCUSSION THREAD

**Discussion by atp38** [acheong08/EdgeGPT#389] [Q&A]
What is the best way to clear the conversation? 
At the moment I am using this line to ask questions:

await bot.ask(prompt=Question, conversation_style=ConversationStyle.creative, wss_link="wss://sydney.bing.com/sydney/ChatHub"))

I would like to clear the conversation in-between each question.

note: there is a similar discussion already but the only answer was to use the chat gpt api. That is a workable answer but being able to use bing chat would be better for me

Thank you for reading :)

**Comment by acheong08** [comment]
bot.reset()
